the_debuginator.h

```diff
@@ -2377,6 +2377,30 @@ void debuginator_move_to_parent(TheDebuginator* debuginator) {
 	}
 }
 
+void debuginator_move_to_root(TheDebuginator* debuginator) {
+	DebuginatorItem* hot_item = debuginator->hot_item;
+	while (hot_item && hot_item != debuginator->root) {
```

Comment:
Apparently github did not actually save my comments, it just pretended to, so here it goes again:

The reason for the while loop is folders-in-folders; we wanted behavior to be consistent when you expand something, so if I'm in a leaf I want to expand all folders leading back to the root. That way when I expand the first child again all its children are collapsed.